service/controller/clusternamespace/clusternamespace_resources.go

```diff
@@ -32,8 +35,36 @@ func newClusterNamespaceResources(config clusterNamespaceResourcesConfig) ([]res
 		}
 	}
 
+	var rbacCleanerResource resource.Interface
+	{
+		c := rbaccleaner.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+		}
+
+		rbacCleanerResource, err = rbaccleaner.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var rbacAppOperatorResource resource.Interface
+	{
+		c := rbacappoperator.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
+		}
+
+		rbacAppOperatorResource, err = rbacappoperator.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
 	resources := []resource.Interface{
 		clusterNamespaceResourcesResource,
+		rbacCleanerResource,
+		rbacAppOperatorResource,
```

Comment:
nit: sort

Reply:
@kubasobon What do you mean? Is the updated formatting / order better / what you intended?

Reply:
Ah, the resources were out of alphabetical order, but it's a microscopically tiny nit. Feel free to ignore :D